elementQuery.js

```diff
@@ -164,6 +164,12 @@
                             processSelector(rule[cssRules][j].selectorText);
                         }
                     }
+                    // Handle @imported stylesheets
+                    else if (rule.styleSheet && rule.styleSheet.cssRules && rule.styleSheet.cssRules.length) {
```

Comment:
Any reason this is in an <code>else if</code>? Wouldn't it be possible that a stylesheet has some CSS rules and an import?